README.md

```diff
@@ -45,6 +45,18 @@ cargo install rustlings
 
 </details>
 
+> [!CAUTION]
```

Comment:
I've used caution but feel free to nit to warning or [any other](https://github.com/orgs/community/discussions/16925).